dropwizard/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaSettings.java

```diff
@@ -0,0 +1,955 @@
+         */
+        Ssl setKeyStoreProvider(String keyStoreProvider) {
+            this.keyStoreProvider = keyStoreProvider;
+            return this;
+        }
+
+        /**
+         * Returns the trust store that holds SSL certificates.
+         * @return the trust store
+         */
+        @Nullable
+        String getTrustStore() {
+            return trustStore;
+        }
+
+        /**
+         * Sets the trust store that holds SSL certificates.
+         */
+        Ssl setTrustStore(String trustStore) {
+            this.trustStore = trustStore;
+            return this;
+        }
+
+        /**
+         * Returns the password used to access the trust store.
+         * @return the trust store password
+         */
+        @Nullable
+        String getTrustStorePassword() {
+            return trustStorePassword;
+        }
+
+        /**
+         * Sets the password used to access the trust store.
+         */
+        Ssl setTrustStorePassword(String trustStorePassword) {
+            this.trustStorePassword = trustStorePassword;
+            return this;
+        }
+
+        /**
+         * Returns the type of the trust store.
+         * @return the trust store type
+         */
+        @Nullable
+        String getTrustStoreType() {
+            return trustStoreType;
+        }
+
+        /**
+         * Sets the type of the trust store.
+         */
+        Ssl setTrustStoreType(String trustStoreType) {
+            this.trustStoreType = trustStoreType;
+            return this;
+        }
+
+        /**
+         * Returns the provider for the trust store.
+         * @return the trust store provider
+         */
+        @Nullable
+        String getTrustStoreProvider() {
+            return trustStoreProvider;
+        }
+
+        /**
+         * Sets the provider for the trust store.
+         */
+        Ssl setTrustStoreProvider(String trustStoreProvider) {
+            this.trustStoreProvider = trustStoreProvider;
+            return this;
+        }
+    }
+
+    /**
+     * Configurations for the HTTP content encoding.
+     */
+    static class Compression {
+        /**
+         * Specifies whether the HTTP content encoding is enabled.
+         */
+        private boolean enabled;
+
+        /**
+         * The MIME Types of an HTTP response which are applicable for the HTTP content encoding.
+         */
+        private String[] mimeTypes = {
+                "text/html", "text/xml", "text/plain", "text/css", "text/javascript",
+                "application/javascript", "application/json", "application/xml"
+        };
+
+        /**
+         * The {@code "user-agent"} header values which are not applicable for the HTTP content encoding.
+         */
+        @Nullable
+        private String[] excludedUserAgents;
+
+        /**
+         * The minimum bytes for encoding the content of an HTTP response.
+         */
+        @Nullable
+        private String minResponseSize;
```

Comment:
nit: fields should be at top of class

Reply:
I wonder why do you think this move to at top of the class? I guess because this is an important property than others?